Google-Summer-of-Code-2023.md

```diff
@@ -273,3 +273,504 @@ The following is a list of Oppia's 2023 GSoC project ideas. You are welcome to c
+
+### 1.3. Serial chapter launch
+
+TBA
+
+### 1.4. User feedback reporting UI on Web
+
+TBA
+
+### 1.5. Learner dashboard redesign
+
+TBA
+
+### 1.6. Improve math interactions
+
+TBA
+
+### 1.7. Accessibility improvements
+
+TBA
+
+## Contributor Dashboard team
+
+### 2.1. Contributor Dashboard Admin Stats Table
+
+**Project Description:**
+
+Contributor dashboard admin page revamp. Table view of contributors with key stats, e.g. number of translations/questions submitted/reviewed. Each row represents a contributor and is actionable, allowing an admin to add or remove contribution rights. 
+
+**Size of this project:** medium (~175 hours)
+
+**Potential Mentors:** 
+
+**Knowledge/Skills Recommended:**
+
+* Knowledge and understanding of Python
+* Knowledge and understanding of TypeScript, Angular, basic HTML, and CSS
+
+**Suggested Milestones:**
+
+* **Milestone 1:** Complete the complete backend changes:
+  - Introduce language coordinator admin role
+  - Modify contributor dashboard admin controller to allow adding or removing language coordinators
+  - Add new endpoint to contributor dashboard admin controller for fetching contributor stats
+
+* **Milestone 2:** Complete the frontend changes:
+  - Display contributor stats table on contributor dashboard admin page
+  - Show different available admin actions depending on user admin role
+  - Add support for sorting by table column (will require some backend query changes)
+  - Implement table pagination
+
+
+**Dependency on Release Schedule:** 
+
+**Proposal notes:**
+
+* For contributor stats, you can query the existing stats models in core/storage/suggestion/gae_models.py, e.g. `TranslationContributionStatsModel`. 
+* To support sorting by table column, we’ll need to add a sort key for each attribute. The sort key will be used in the storage query. See PR [#1653](https://github.com/oppia/oppia/pull/16534) for an example. 
+* Pagination is hard. See PR [#16289](https://github.com/oppia/oppia/pull/16289) to see how pagination was added to the existing contributor stats tables.
+
+
+**Useful resources:**
+
+### 2.2. Prioritized Translation Languages and Auto-Translations
+
+**Project Description:**
+
+Support the marking of languages as “prioritized” for translation. Prioritized languages will have their remaining untranslated lesson text content auto-translated to make the lessons available in the language as quickly as possible. Auto-translated content will appear with a note explaining the auto-translation. Non-prioritized languages will also be auto-translated after a threshold of translation, e.g. 95%. 
+
+Additionally, finish adding [computer-aided translation (CAT)](https://docs.google.com/document/d/1kJd-yLTzB9a2c3Nq7v9pzKfHwKHKGpkWfQ8B0YGf50U/edit#heading=h.24wysknhgyrz) support when submitting translation suggestions. 
+
+**Size of this project:** medium (~175 hours)
+
+**Potential Mentors:** 
```

Comment:
How about putting "TBA" for the mentors, again to make clear that we didn't forget about this section?